.github/dependabot.yml

```diff
@@ -52,82 +52,82 @@ updates:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/dynamic-config-cp"
+  directory: "/examples/ext_authz"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/ext_authz"
+  directory: "/examples/fault-injection"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/ext_authz/auth/http-service"
+  directory: "/examples/grpc-bridge"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/ext_authz/auth/grpc-service"
+  directory: "/examples/kafka"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/fault-injection"
+  directory: "/examples/local_ratelimit"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/grpc-bridge"
+  directory: "/examples/mysql"
   schedule:
     interval: daily
 
 - package-ecosystem: "docker"
-  directory: "/examples/grpc-bridge/client"
```

Comment:
am I failing at search or did we lose this one?

Reply:
we moved the recipes into shared folders to limit the dependabot activity, improve docker layering and make maintenance easier see here https://github.com/envoyproxy/envoy/pull/26005/files